qgis_notebook/dialogs/notebook_dock.py

```
@@ -1571,6 +1614,56 @@ def _load_theme(self):
+        # Pick up values written by the (separate) settings dock's QSettings.
+        self.settings.sync()
+        self._load_theme()
+        self._setup_ui()
+        if old_widget is not None:
+            old_widget.deleteLater()
+
+        # _setup_ui() shows the welcome screen; re-render the notebook if one
+        # is loaded and restore each cell's captured output.
+        if self.notebook_data and self.notebook_data.get("cells"):
+            self._render_notebook()
+            for widget, output in zip(self.cell_widgets, outputs):
```

Ruff (0.15.21)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)